### ISSUE TITLE
0 size file cannot be `get`ed

### ISSUE THREAD

**Issue by yarikoptic** [DanielDent/git-annex-remote-rclone#46]
I think it is git-annex issue. Filed https://git-annex.branchable.com/bugs/error_out_on_file_size_0_from_external_spec_remote/ . Testing for that would be added in #45 (but disabled)

**Comment by yarikoptic** [comment]
regression in git annex introduced in [8.20201007-86-gdad4be97c AKA 8.20201103~60](https://git.kitenet.net/index.cgi/git-annex.git/commit/?id=dad4be97c2057db1ef3a13bb983d1701a90c9069),
said to be fixed in [10.20220525-73-g13fc6a9b6](https://git.kitenet.net/index.cgi/git-annex.git/commit/?id=13fc6a9b6ad4f0ee0a783c91825ab7ad35dd2b64)

**Comment by yarikoptic** [comment]
they seems can! i.e. there is nothing to fix in here seems, since tests in #52 passed for me locally... I am a little confused then since I was expecting failure due to checking at https://github.com/DanielDent/git-annex-remote-rclone/blob/master/git-annex-remote-rclone#L220 IMHO erroneously that size is not 0.  Oh well, it doesn't match for me since rclone output format has changed:

```
$> HOME=$PWD/.. rclone size local:/home/yoh/.tmp/dl-QuTTG2I/rclone-local/f87/4d5/SHA256E-s0-S104857600-C1--e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
Total objects: 1 (1)
Total size: 0 B (0 Byte)

$> rclone --version
rclone v1.59.0-beta.6202.cfe0911e0
```

so the test was passing because we have no chance to match that 0 ;)  I think that check is irrelevant and erroneous so I will suggest to remove it within that PR.

**Comment by yarikoptic** [comment]
ok, that PR was merged and the check for 0 was removed, we should be all good.
